DataFormats/simulation/include/SimulationDataFormat/MCTruthContainer.h

```diff
@@ -64,6 +65,13 @@ class MCTruthContainer : public TNamed
   size_t getIndexedSize() const { return mHeaderArray.size(); }
   // return the number of elements managed in this container
   size_t getNElements() const { return mTruthArray.size(); }
+
+  // get individual "view" container for a given data index
+  // the caller can do modifications on this view (such as sorting)
+  gsl::span<TruthElement> getLabels(int dataindex) {
+    return gsl::span<TruthElement>(&mTruthArray[mHeaderArray[dataindex].index], mHeaderArray[dataindex].size);
+  }
+
```

Comment:
Shouldn't this function be const?

Reply:
Not necessarily. This particular version would allow to modify the view. A const version will be added in addition.